src/Raven.Client/Documents/Operations/CompareExchange/DeleteCompareExchangeValueOperation.cs

```diff
@@ -7,11 +7,24 @@
 
 namespace Raven.Client.Documents.Operations.CompareExchange
 {
+    /// <summary>
+    /// Operation to delete a compare exchange value in a RavenDB database. 
+    /// A compare exchange value is a key-value pair that is part of the RavenDB distributed coordination mechanism.
```

Comment:
instead of 
> part of the RavenDB distributed coordination mechanism

maybe something like 
> A compare exchange is a key-value pair is a distributed coordination mechanism that ensures a consistent state across the cluster

Reply:
Done